site/experimental-features.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Experimental features
+layout: default
+---
+
+# Experimental features
+
+ * [Proxy](#proxy)
+
+### <a name="proxy"></a>Proxy
+
+Instead of the weave command-line utility, you may prefer to use the standard
+docker command-line utility to manage containers. This will allow containers to
+be started in foreground mode. It will also ensure that the weave network
+interface is available before running your container. However, it does not
+currently support TLS.
+
+#### Setup
+
+To do this, while still connecting containers to the weave network, you can use
+the weave proxy. We can start the proxy with
+
+    host1$ weave launch-proxy
+
+By default, the proxy will connect to docker at `unix:///var/run/docker.sock`, and listen
+on port 12375. However, we can adjust these via the `-H` and `-L` arguments,
+respectively. All docker commands can be run via the proxy, so it is safe to
+globally adjust your `DOCKER_HOST`.
+
+    host1$ export DOCKER_HOST="tcp://host1:12375"
+    host1$ export DOCKER_TLS_VERIFY=0
```

Comment:
I'd leave that out. If the user ordinarily connects to docker via tls, turning that off is not a good idea; it's better for that to fail.